bin/yaml/dotnet.yaml

```diff
@@ -8,7 +8,7 @@ compilers:
       - name: trunk
         if: nightly
         type: nightly
+      - v6.0.11
+      - v6.0.14
       - v7.0.1
-      - name: v6.0.11
```

Comment:
Does 6.0.11 no longer need this environment variable set to run?

Reply:
it is now set in https://github.com/compiler-explorer/compiler-explorer/blob/60d2f8fa5f02392be84990bcf947aae090d01303/lib/compilers/dotnet.ts

Reply:
This yaml does NOT use that compiler-explorer script, the `ce_install` command requires the `--version` to see if the dotnet executable is there, are you sure that will work without the environment variable? have you tested it?

Reply:
> have you tested it?

no but ci did https://github.com/compiler-explorer/infra/actions/runs/4195904213 and it's green.

Reply:
The CI cannot test that. You can test it by following the instructions on the readme https://github.com/compiler-explorer/infra/blob/main/README.md

Reply:
tested, dotnet --version is working without this environment variable

Reply:
Cool, thanks